src/storm/solver/TopologicalMinMaxLinearEquationSolver.cpp

```diff
@@ -151,6 +151,7 @@ bool TopologicalMinMaxLinearEquationSolver<ValueType, SolutionType>::internalSol
         }
 
         // If requested, we store the scheduler for retrieval.
+        // TODO when is this code called??????? im a bit confused
```

Comment:
@tquatmann

Reply:
When we want to produce an optimal scheduler for the given MinMax problem, the `trackScheduler` flag will be set to true. 
The code below is doing a single iteration of value iteration to find out the optimal choices.